common/djangoapps/student/views.py

```diff
@@ -1053,7 +1053,9 @@ def login_user(request, error=""):  # pylint: disable=too-many-statements,unused
     running_pipeline = None
     third_party_auth_requested = third_party_auth.is_enabled() and pipeline.running(request)
     third_party_auth_successful = False
-    trumped_by_first_party_auth = bool(request.POST.get('email')) or bool(request.POST.get('password'))
+    trumped_by_first_party_auth = bool(
```

Comment:
Nit: Could use any() here. Not sure if it would be the better way to do it or not.